dev-resources/template/marketplace/0_vpc.yml

```diff
@@ -0,0 +1,224 @@
+AWSTemplateFormatVersion: '2010-09-09'
```

Comment:
should i assume this is identical to old vpc? if so do we need the dupe?

Reply:
it is identical I just duped for consistency, but duping here isn't necessary.